org.kde.krita.yaml

```diff
@@ -865,10 +878,8 @@ modules:
           project-id: 1477
           stable-only: true
           url-template: https://s3.amazonaws.com/json-c_releases/releases/json-c-$version.tar.gz
+  - shared-modules/intltool/intltool-0.51.json
   - name: libmypaint
-    buildsystem: cmake-ninja
```

Comment:
Any reason we're not using cmake anymore here?

Reply:
it's faster? the rule is if cmake-ninja work, use it. (it's `cmake -G Ninja`)

Reply:
libmypaint doesn't support building using cmake without the custom patches that Krita added here https://invent.kde.org/graphics/krita/-/tree/master/3rdparty/ext_mypaint. Would it be better to build it without those patches or keep the patches in to use cmake-ninja?

Reply:
Hum, there is unfortunately no good options here:
- If we don't use the patches then we deviate from how upstream Krita tests & builds
- If we use the patches, we deviate from how upstream libmypaint tests & builds
Given that Krita is already a snowflake, I'd say we keep doing as they do upstream?
WDYT?

Reply:
Sure, going ahead and readding the patches for cmake-ninja.